zlibWrapper/Makefile

```diff
@@ -7,8 +7,9 @@
 
 # Paths to static and dynamic zlib and zstd libraries
 ifneq (,$(filter Windows%,$(OS)))
-STATICLIB = ../../zlib/libz.a ../lib/libzstd.a
-IMPLIB    = ../../zlib/libz.dll.a ../lib/libzstd.a
+ZLIBDIR = ../../zlib1.2.8
```

Comment:
`ZLIBDIR` : how is this path determined ?
If it must be set manually, and if it is dependent on local settings,
could there be a message and/or a comment to attract user attention on the need to define this variable (potentially directly on the command line) ?